demo/index.html

```diff
@@ -111,8 +110,8 @@
 
       <paper-dialog id="dropdownDialog">
         <h2>Dialog Title</h2>
-        <paper-dropdown-menu label="Value">
-          <paper-menu class="dropdown-content">
+        <paper-dropdown-menu-light label="Value">
+          <paper-listbox class="dropdown-content" slot="dropdown-content">
```

Comment:
`paper-dropdown-menu-light 1.x` requires `class="dropdown-content"` to distribute this listbox (1.x variant), whereas `paper-dropdown-menu-light 2.x` requires `slot="dropdown-content"` (main variant)